packages/plugins/plugin-thread/src/components/command-extension.ts

```diff
@@ -27,10 +27,7 @@ const parser = StreamLanguage.define<{ count: number }>({
 });
 
 const styles = HighlightStyle.define([
-  {
-    tag: tags.tagName,
-    class: mx(tagRoot({ palette: 'cyan' }), 'text-baseText font-medium'),
-  },
+  { tag: tags.tagName, class: mx('[data-hue="indigo"] dx-tag text-baseText font-medium') },
```

Comment:
`[data-hue="indigo"]` isn’t a classname on its own, usually it should be something like `data-[hue="indigo"]:something`.

Reply:
Hmmm, in this case I want to *set* the data attribute...

Reply:
Can I do something like `[&[data-hue='indigo']]`?

Reply:
I'm not sure how to actually exercise this code in composer or the storybooks right now either...

Reply:
I think there's a story for the slash commands but if you can't find it, ask @richburdon

Reply:
Found it in composer @wittjosiah. Thanks.